components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Runtime/Default/SecurityConfig.cs

```diff
@@ -45,19 +50,19 @@ public class SecurityConfig : ConfigBase, ISecurityConfig {
 
         /// <inheritdoc/>
         public CertificateStore TrustedIssuerCertificates => new CertificateStore {
-            StorePath = GetStringOrDefault(TrustedIssuerCertificatesPathKey, () => "/pki/trusted"),
```

Comment:
>rusted [](start = 90, length = 6)

issuers, or you have a security issue...

Reply:
I left it initially unchanged, but it makes sense to fix it